e2e/lib/k8up.bash

```diff
@@ -20,18 +24,23 @@ setup() {
 
 setup_file() {
 	reset_debug
+	clear_pv_data
 }
 
 teardown() {
 	cp -r /tmp/detik debug || true
 }
 
+clear_pv_data() {
+	rm -rfv ./debug/data/*
```

Comment:
this doesn't work, because this also deletes `./debug/data/pvc-subject`, which is set up by `make` with our local user id. If we remove `pvc-subject`, then KIND will create this path when a PV is created, but with root permissions.

This cascades to the point where the subject deployment cannot create the test file with tee:
`tee: /data/old_file.txt: Permission denied`
and it crashloops.

you can easily reproduce this with `make clean e2e-test`